### PR TITLE
Use distro config to change data dir

### PR DESCRIPTION
**Pull request by razzeee**

### CODE REVIEW

**Comment by razzeee** [comment]
@faveoled can you sanity check this?

**Comment by faveoled** [comment]
Says this in About page which I think is incorrect for Flatpak:
```
DATA_DIR = "/home/user/.pgadmin"
```

**Comment by razzeee** [comment]
Should be 
image

Is
image

Not really sure why

**Comment by razzeee** [comment]
Should be fixed

**Comment by faveoled** [comment]
Looks good
image

File: org.pgadmin.pgadmin4.yml [hunk 1]
```diff
@@ -100,12 +100,14 @@ modules:
       - tar -xf data.tar.zst
       - rm -rf usr/pgadmin4/venv
       - cp -r usr/* /app/
-      - sed -i "233 i DATA_DIR = os.environ['XDG_CONFIG_HOME']" /app/pgadmin4/web/config.py
+      - cat config_distro.py usr/pgadmin4/web/config_distro.py >> ${FLATPAK_DEST}/pgadmin4/web/config_distro.py
```
Comment (faveoled) [inline org.pgadmin.pgadmin4.yml]: Doesn't it duplicate existing config_distro.py content? Two files in cat + append to existing

Comment (faveoled) [reply org.pgadmin.pgadmin4.yml]: It's duplicated:
```
$ cat /var/lib/flatpak/app/org.pgadmin.pgadmin4/current/active/files/pgadmin4/web/config_distro.py 
HELP_PATH = '../../../share/docs/en_US/html/'
DEFAULT_BINARY_PATHS = {
    'pg':   '/usr/bin',
    'ppas': ''
}
import os

DATA_DIR = os.environ["XDG_CONFIG_HOME"]
HELP_PATH = '../../../share/docs/en_US/html/'
DEFAULT_BINARY_PATHS = {
    'pg':   '/usr/bin',
    'ppas': ''
}
```

Comment (razzeee) [reply org.pgadmin.pgadmin4.yml]: Should be better

**Comment by faveoled** [comment]
config_distro.py with no .yml changes? Not sure how it's supposed to work

**Comment by razzeee** [comment]
Probably messed up my push, will check in the evening - sorry

**Comment by razzeee** [comment]
Should be better

**Comment by faveoled** [comment]
Yeah. works fine

**Pull request merged**